### DISCUSSION TITLE
Is there a way to configure AccessibilityInfo.isScreenReaderEnabled to return false?

### DISCUSSION THREAD

**Discussion by mr-snow2** [necolas/react-native-web#2072] [Help] [answered]
Is there a way to configure isScreenReaderEnabled to return false?
Is it intentional that it is always true or just since it was the better default?

**Comment by necolas** [comment, marked as answer]
It's always true on web and there is no web-native api equivalent

**Reply by mr-snow2** [reply]
Thanks for the quick answer.
Is there a plan to have it configurable?
The reason I'm asking is that some libs (in my case ExpandableCalendar from react-native-calendars) show different components more suitable for screen reader when it is enabled. In this case I cannot use some of their components directly.
I'm not sure what is the best solution.
